ui/packages/docs/src/components/AoE/character_data.json

```diff
@@ -1073,6 +1073,90 @@
       }
     ]
   },
+  "charlotte": {
+    "cons": [
+      {
+        "ability": "C6 Heal",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 3.5
+      },
+      {
+        "ability": "C6 Attack",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 5
+      }
```

Comment:
replace this with
```
{
    "ability": "C6",
    "shape": "Circle", 
    "center": "GlobalValue",
    "radius": 3.5,
    "notes": "Spawns on target that the C6 proc'd on. Heal is 5m instead of 3m.",
}
```